test/src/unit-cppapi-webp-filter.cc

```diff
@@ -358,16 +256,24 @@ TEMPLATE_LIST_TEST_CASE(
     auto invalid_attr = Attribute::create<TestType>(ctx, "rgb");
     invalid_attr.set_filter_list(filterList);
 
-    // WebP filter requires at least 2 dimensions.
+    // WebP filter requires exactly 2 dimensions for Y, X.
     {
       Domain invalid_domain(ctx);
       invalid_domain.add_dimension(
           Dimension::create<uint64_t>(ctx, "y", {{1, 100}}, 90));
 
+      // Test with < 2 dimensions.
       ArraySchema invalid_schema(ctx, TILEDB_DENSE);
       invalid_schema.set_domain(invalid_domain);
       invalid_schema.add_attribute(valid_attr);
+      REQUIRE_THROWS_AS(
+          Array::create(webp_array_name, invalid_schema), tiledb::TileDBError);
```

Comment:
We could do extra validation to make sure we get the exact exception we expect here.